cit-api/pipeline/models/approval_status.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib.gis.db import models
+
+class ApprovalStatus(models.Model):
+    status_name = models.CharField(max_length=255, null=False)
```

Comment:
I think for this model we probably want a slightly different structure. it should have an auto incrementing ID, a status code, status name, description of the status, and a boolean indicator of if it is an active status.

Reply:
Sounds good, I'll update this